robolectric/src/test/java/org/robolectric/shadows/ShadowLocaleDataTest.java

```diff
@@ -12,6 +12,11 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RunWith(TestRunners.MultiApiWithDefaults.class)
+@Config(sdk = {
```

Comment:
Maybe these can be in another PR if they pass happily. Lets keep this one focused.